assets/css/_mixins.scss

```diff
@@ -1,4 +1,6 @@
 $orange: #f5914f;
+$grey: rgba(122, 120, 120, 1);
+$borderRadius3: 3px;
```

Comment:
I'm not sure this is good variable name. Consider renaming to `$defaultBorderRadius`, `$formInputBorderRadius`.

Why? Because sometimes you need to change the value. And having `$borderRadius3: 4px;` will look odd and will force you to rename variable. Which means this name is bad